packages/client/modules/meeting/components/AddNewPokerTemplate.tsx

```diff
@@ -74,16 +110,10 @@ const AddNewPokerTemplate = (props: Props) => {
     <div>
       {error && <ErrorLine>{error.message}</ErrorLine>}
       <AddPokerTemplateLink palette='blue' onClick={addNewTemplate} waiting={submitting}>
-        {'Create New Template'}
+        {`Create New Template ${featureFlags.templateLimit && tier === 'personal' && '🔒'}`}
```

Comment:
-1 This is not what I had suggested and will produce "Create New Template false". You're doing string interpolation here which will print the result of `${}`. I had suggested to move this one level up (i.e. remove the surrounding {} and backticks), then React would take the `false` component and ignore it. Either way, now it's broken.